src/util/VanDerCorput.cpp

```diff
@@ -116,10 +131,8 @@ VanDerCorput::const_iterator VanDerCorput::end() const
 }
 
 //=============================================================================
-VanDerCorput::const_iterator::const_iterator(const VanDerCorput *seq)
-    : mSeq(seq)
-    , mN(-1)
-    , mFinalIter(false)
+VanDerCorput::const_iterator::const_iterator(const VanDerCorput* seq)
+  : mSeq(seq), mN(-1), mFinalIter(false)
```

Comment:
Okay, let's turn on `BreakConstructorInitializersBeforeComma` option to enforce putting commas at the beginnings of every initializer.



> Also, this is exactly why I prefer to spend time hand formatting the code to pass the check  rather than spending too much time trying to make the format operation idempotent: there are going to be many issues like this where we want something that the formatter won't do, as well as the other issue with clang 3.9's formatter just churning the code for no reason.

I basically agree with you. Alternatively, we could try other formatter that allows us much more fine tuning, but it would also be time-consuming work to find the best configuration for us. But I also think we don't want to repeat the same work of hand formatting for every PR from various contributors and we could be inconsistent.

Let's give a try auto formatting with clang-format and see if we really need to find other solution. :smile: